src/composables/discord/notifyDiscordSale.ts

```diff
@@ -0,0 +1,112 @@
+import { PNFT } from "../../common/helpers/types";
+import Discord, { MessageEmbed, TextChannel } from "discord.js";
+
+
+
+export function generateTicketDetailLink (ticket: PNFT) {
```

Comment:
todo: after validating move-pinata-ticket-functions branch changes, merge to this one and reduce duplicated function usage